templates/index.html

```diff
@@ -15,48 +15,59 @@
 <html>
 <head>
     <title>MAX Question Answering Chatbot Demo</title>
-    <link rel="stylesheet" href="static/css/bootstrap.min.css">
+    <link rel="stylesheet" href="static/css/bootstrap.css">
```

Comment:
What is the reason for this change?

Reply:
I updated to the latest version of Boostrap (4), and I prefer the non-minimized version cause it's easier to debug and alter for other people.. Given the scale that we work at the loading perfomance is negligible